services/movie_session.py

```diff
@@ -0,0 +1,43 @@
+from db.models import MovieSession, Movie, CinemaHall
+
+
+def create_movie_session(movie_show_time: str,
+                         movie_id: int,
+                         cinema_hall_id: int) -> MovieSession:
+    movie = Movie.objects.get(id=movie_id)
+    cinema_hall = CinemaHall.objects.get(id=cinema_hall_id)

```

Comment:
The `create_movie_session` function does not handle the case where the movie or cinema hall with the given IDs do not exist. Consider using try-except blocks to handle `Movie.DoesNotExist` and `CinemaHall.DoesNotExist` exceptions.